appveyor.yml

```diff
@@ -0,0 +1,49 @@
+version: 1.0.0_b{build}-{branch}
+
+os: Visual Studio 2015
+
+environment:
+  TERM: dumb
+  COVERALLS_REPO_TOKEN:
+    secure: 5nuCg+faxFPeppoNNcSwVobswAVFUf8ut83vw8CX/4W2y0kZkGmwEfCUxSQWiQDU
+  CI_NAME: appveyor
+  CI_JOB_ID: $(APPVEYOR_JOB_ID)
+  GCOV_PREFIX: $(APPVEYOR_BUILD_FOLDER)
+  matrix:
+  - COMPILER: msvc
+    GENERATOR: "Visual Studio 14 2015"
+    CFLAGS: /MP
+    CXXFLAGS: /MP
+  - COMPILER: mingw
+    GENERATOR: "MSYS Makefiles"
+
+platform:
+  - x86
+  - x64
+
+configuration:
+  - Debug
+  - Release
+  - RelWithDebInfo
+
+init:
+  - set PATH=C:\msys64\mingw64\bin;C:\msys64\usr\bin;C:\Python35;C:\Python35\Scripts;%PATH%
+  - set MSYSTEM=MINGW64
+  - pip install cram==0.7
+
+install:
+  - mkdir build && cd build
+  - >
+      cmake
+      -Wno-dev
+      -DCMAKE_INSTALL_PREFIX=boxfort-%RELEASE_NAME%
+      -DCMAKE_BUILD_TYPE="%CONFIGURATION%"
+      -DBXF_STATIC_LIB=ON ..
```

Comment:
Without this the build fails as if compiling with shared lib is not possible. The linker looks for boxfort.lib even in case of shared build.